### DISCUSSION TITLE
扩展一个新端之前，如何判断是否能够支持taro运行时能力？

### DISCUSSION THREAD

**Discussion by broven** [NervJS/taro#11511] [Q & A] [answered]
最近微博也开始出小程序了， 我们想调研看看我们基于Taro3.x的代码能否移植到微博小程序上，在开始开发端扩展模块之前， 有没有哪些能力是一定需要端上具备的？

**Comment by Chen-jj** [comment, marked as answer]
按经验来说，template 的兼容性是最必要的。也就是 `dist/base.wxml` 里的语法新端都支持的话就问题不大。
